easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.2-foss-2016a-R-3.2.3.eb

```diff
@@ -80,7 +80,7 @@ exts_list = [
     ('DEGseq', '1.24.0', bioconductor_options),
     ('hgu133plus2.db', '3.2.2', bioconductor_options),
     ('illuminaio', '0.12.0', bioconductor_options),
-    ('rtracklayer', '1.30.2', bioconductor_options),
+    ('rtracklayer', '1.30.3', bioconductor_options),
```

Comment:
already updated to 1.30.4 by now...

We should probably do another full update round for all of these packages?